src/helpers/ContractSynchronizer.ts

```diff
@@ -1,15 +1,37 @@
-import getOwnedERC721 from 'helpers/getOwnedERC721'
+import { ContractReceipt } from 'ethers'
+import getOwnedERC721, {
+  isTransferEvent,
+  parseLogData,
+} from 'helpers/getOwnedERC721'
 
 export default class ContractSynchronizer {
   account: string
   locked = false
   synchronizedBlockId?: number
   addressToTokenIds: { [address: string]: Set<string> } = {}
+  skipTransactions = new Set<string>()
```

Comment:
I don't think we will have a variety of hashes. Do we really need `Set`? If so, okay then

Reply:
It would be possible to replace this with an array, I'll think about it